src/main/java/com/google/devtools/build/lib/syntax/FuncallExpression.java

```diff
@@ -745,11 +745,10 @@ private Object invokeObjectMethod(
       throws EvalException, InterruptedException {
     Location location = call.getLocation();
     @Nullable
-    Object fieldValue =
-        (value instanceof ClassObject) ? ((ClassObject) value).getValue(method) : null;
+    Object fieldValue;
     if (function != null) {
       return function.call(positionalArgs, keyWordArgs, call, env);
-    } else if (fieldValue != null) {
+    } else if ((fieldValue = tryGetFieldValue(value, method)) != null) {
```

Comment:
To make the code more readable:
 - remove the `else` (the `if` branch has a return`)
 - declare fieldValue with the initialization (no need for a separate function, inline it like it was)
 - do the `if` after that

Reply:
excellent suggestion!